lib/cli/ui/prompt.rb

```diff
@@ -3,22 +3,7 @@
 # typed: true
 
 require 'cli/ui'
-require 'readline'
-
-module Readline
-  unless const_defined?(:FILENAME_COMPLETION_PROC)
```

Comment:
`reline` [always defines this constant](https://github.com/ruby/reline/blob/4ce247ce2b0965ef25b9d8c7f5c712e73d7e433e/lib/reline.rb#L15).